content/about/security-release-process/security-release-process.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Security Release Process
+description: Release process for Istio security vulnerabilities.
+weight: 35
+page_icon: /img/security-release.svg
+---
+
+Istio follows the following process when releasing patches for critical
```

Comment:
The Istio team follow the process below when releasing...

Reply:
Done.